src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        ArrayList<Task> taskList = new ArrayList<Task>();
+        String taskName = "";
+        Scanner sc = new Scanner(System.in);
+        System.out.println("\t____________________________________________________________\r\n"
+                + "\t Hello! I'm Duke\r\n"
+                + "\t What can I do for you?\r\n"
+                + "\t____________________________________________________________\r\n");
+
+        while (!taskName.equals("bye")) {
+            taskName = sc.next();
+
+            if (!taskName.equals("bye")) {
```

Comment:
Is this possibly a redundant check? Seems to be the same as the while condition just above